Project_Xcavate.md

```diff
@@ -0,0 +1,158 @@
+Alex Ward – 25 years in the global property market – serial Entrepreneur
+Akash Takyar - Akash has a master’s in computer application with a deep understanding of IoT and Blockchain. Akash is a world-renowned expert, blockchain influencer, and technical architect.
+Deepak Shokeen - With over 20 years of experience engineering and developing apps. He has a deep understanding of building Cloud, IoT, and Blockchain applications.
+Kapil Saxena - Kapil is a design professional with 15+ years of experience in User Experience & User Interface design. Currently, he is leading
+the design team at LeewayHertz designing UX and UI for Blockchain Applications, handheld devices, Artificial Intelligence Solutions, and IoT Products.
+
+### Team Code Repos
+
+https://github.com/xcavate2022
+https://github.com/LeewayHertz
+https://github.com/orgs/XDCFoundation/repositories
+https://github.com/XDCFoundation/XDC3Android
+https://github.com/XDCFoundation/xdc-testnetwork-leo
+
+### Team LinkedIn Profiles
+
+https://www.linkedin.com/in/richard-houldsworth/ 
+https://www.linkedin.com/in/alex-ward-2350a036/
+https://www.linkedin.com/company/leewayhertz-technologies/
+
+## Development Status :open_book:
+
+We have been heavily involved in learning all things associated with the Polkadot & Kusama Ecosystem. This has been coupled with the idea of bringing land and property from its current state into the web3 ecosystem. We have been meeting with top-level industry leads, government officials and developers.
+Robin Ejsmond-Frey from Parity has been a tremendous help by supporting us with information about the W3F grant application process and recommendations to attend Hackathons and the Polkadot Decoded Event London. This is where we met Tim Dobie from Parity, who would introduce us to James Ruffer and the web3devs team, plus Paul Castillo from Tokensoft.
+
+We have been taking time to play with Substrate to form initial POCs inside the Polkadot system. We feel we are now ready for the core POC for this project.
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 1 month
+- **Full-Time Equivalent (FTE):**  3
+- **Total Costs:** $36000 USD
+
+### Milestone 1 - Technical & Visual Design and setup
+
+- **Estimated duration:** 1 month
+- **FTE:**  3
+- **Costs:** 36,000 USD
+
+We will start with the user onboarding process to Xcavate parachain/parathread allowing the design team to create low-fi wireframe mockups using Figma that will then be approved by the team to be updated to hi-def mockups that will be interactive for the team's final feedback. This phase will be constant, but the core of the design deliverables will take 3-4 weeks based on feedback. As each of these designs is approved, we will pass them on to the development team to complete in tandem. We will, however, separate these into another milestone. Also, during this first phase, developers will set up the developer's environment with dev env and third-party sandboxes.

```

Comment:
> that will then be approved by the team

Which team? We usually expect at least low-fi designs or wireframes to exist for a UI grant.